invenio_app_ils/ui/src/semantic-ui/site/collections/menu.overrides

```diff
@@ -0,0 +1,3 @@
+/*******************************
+         Site Overrides
+*******************************/
```

Comment:
missing new line :)

Reply:
those files will be cleaned later, and they are copied from semantic